reference/commands/misc/inspect.rst

```diff
@@ -4,17 +4,13 @@
 conan inspect
 =============
 
-.. warning::
```

Comment:
We have to keep this warning. This was introduced manually and it is something not implemented in the script

Reply:
Maybe mark the command as experimental in the script so this message is added automatically?

Reply:
done